tasks/task1/task1_2.es6.js

```diff
@@ -0,0 +1,13 @@
+import fs from 'fs';
+import csv from 'csvtojson';
+import { pipeline } from 'stream';
+import errorHandler from './utils/errorHandler.es6';
```

Comment:
Consider separating your project imports and a third-party imports with a new line.
Think of the huge files with lots of imports - this way your import section will be organised in a more convenient fashion.